lib/utils/ConfigParser.js

```diff
@@ -14,7 +14,7 @@ const DEFAULT_CONFIGS = {
     baseUrl: null,
     waitforTimeout: 1000,
     framework: 'mocha',
-    reporter: 'dot',
+    reporter: [],
```

Comment:
I'm explicitly falling-back to dot reporter here: https://github.com/webdriverio/webdriverio/pull/896/files#diff-ff5050d3c7f51fca90c4691a4599e852R47